[2024-11-23T15:47:20.121+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-23T15:47:20.147+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: interpark_to_s3.extract scheduled__2024-11-21T00:00:00+00:00 [queued]>
[2024-11-23T15:47:20.157+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: interpark_to_s3.extract scheduled__2024-11-21T00:00:00+00:00 [queued]>
[2024-11-23T15:47:20.158+0000] {taskinstance.py:2866} INFO - Starting attempt 9 of 10
[2024-11-23T15:47:20.190+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonVirtualenvOperator): extract> on 2024-11-21 00:00:00+00:00
[2024-11-23T15:47:20.199+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=76) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-23T15:47:20.201+0000] {standard_task_runner.py:72} INFO - Started process 77 to run task
[2024-11-23T15:47:20.202+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'interpark_to_s3', 'extract', 'scheduled__2024-11-21T00:00:00+00:00', '--job-id', '40', '--raw', '--subdir', 'DAGS_FOLDER/test.py', '--cfg-path', '/tmp/tmpp6_3wcbv']
[2024-11-23T15:47:20.203+0000] {standard_task_runner.py:105} INFO - Job 40: Subtask extract
[2024-11-23T15:47:20.266+0000] {task_command.py:467} INFO - Running <TaskInstance: interpark_to_s3.extract scheduled__2024-11-21T00:00:00+00:00 [running]> on host 393e1f4a7c25
[2024-11-23T15:47:20.397+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='interpark_to_s3' AIRFLOW_CTX_TASK_ID='extract' AIRFLOW_CTX_EXECUTION_DATE='2024-11-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='9' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-21T00:00:00+00:00'
[2024-11-23T15:47:20.399+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-23T15:47:20.441+0000] {baseoperator.py:416} WARNING - PythonVirtualenvOperator.execute cannot be called outside TaskInstance!
[2024-11-23T15:47:20.442+0000] {process_utils.py:186} INFO - Executing cmd: /home/***/.local/bin/python -m virtualenv /tmp/venvnwnxf5ad --python=python
[2024-11-23T15:47:20.443+0000] {process_utils.py:190} INFO - Output:
[2024-11-23T15:47:21.971+0000] {process_utils.py:194} INFO - created virtual environment CPython3.12.7.final.0-64 in 708ms
[2024-11-23T15:47:21.972+0000] {process_utils.py:194} INFO -   creator CPython3Posix(dest=/tmp/venvnwnxf5ad, clear=False, no_vcs_ignore=False, global=False)
[2024-11-23T15:47:21.972+0000] {process_utils.py:194} INFO -   seeder FromAppData(download=False, pip=bundle, via=copy, app_data_dir=/home/***/.local/share/virtualenv)
[2024-11-23T15:47:21.973+0000] {process_utils.py:194} INFO -     added seed packages: pip==24.3.1
[2024-11-23T15:47:21.973+0000] {process_utils.py:194} INFO -   activators BashActivator,CShellActivator,FishActivator,NushellActivator,PowerShellActivator,PythonActivator
[2024-11-23T15:47:22.011+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venvnwnxf5ad/bin/pip install -r /tmp/venvnwnxf5ad/requirements.txt
[2024-11-23T15:47:22.012+0000] {process_utils.py:190} INFO - Output:
[2024-11-23T15:47:24.091+0000] {process_utils.py:194} INFO - Collecting git+https://github.com/hahahellooo/interpark.git@0.3/new_crawling (from -r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:24.092+0000] {process_utils.py:194} INFO -   Cloning https://github.com/hahahellooo/interpark.git (to revision 0.3/new_crawling) to /tmp/pip-req-build-u6kdddz2
[2024-11-23T15:47:24.095+0000] {process_utils.py:194} INFO -   Running command git clone --filter=blob:none --quiet https://github.com/hahahellooo/interpark.git /tmp/pip-req-build-u6kdddz2
[2024-11-23T15:47:25.568+0000] {process_utils.py:194} INFO -   Running command git checkout -b 0.3/new_crawling --track origin/0.3/new_crawling
[2024-11-23T15:47:23.336+0000] {process_utils.py:194} INFO -   Switched to a new branch '0.3/new_crawling'
[2024-11-23T15:47:23.337+0000] {process_utils.py:194} INFO -   branch '0.3/new_crawling' set up to track 'origin/0.3/new_crawling'.
[2024-11-23T15:47:23.337+0000] {process_utils.py:194} INFO -   Resolved https://github.com/hahahellooo/interpark.git to commit d2ab75a632be9f578ac04be0e85d884befd10f87
[2024-11-23T15:47:23.345+0000] {process_utils.py:194} INFO -   Installing build dependencies: started
[2024-11-23T15:47:26.178+0000] {process_utils.py:194} INFO -   Installing build dependencies: finished with status 'done'
[2024-11-23T15:47:26.180+0000] {process_utils.py:194} INFO -   Getting requirements to build wheel: started
[2024-11-23T15:47:27.044+0000] {process_utils.py:194} INFO -   Getting requirements to build wheel: finished with status 'done'
[2024-11-23T15:47:27.045+0000] {process_utils.py:194} INFO -   Preparing metadata (pyproject.toml): started
[2024-11-23T15:47:27.937+0000] {process_utils.py:194} INFO -   Preparing metadata (pyproject.toml): finished with status 'done'
[2024-11-23T15:47:28.081+0000] {process_utils.py:194} INFO - Collecting selenium>=4.26.1 (from interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:28.129+0000] {process_utils.py:194} INFO -   Downloading selenium-4.26.1-py3-none-any.whl.metadata (7.1 kB)
[2024-11-23T15:47:28.211+0000] {process_utils.py:194} INFO - Collecting beautifulsoup4>=4.12.3 (from interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:28.221+0000] {process_utils.py:194} INFO -   Downloading beautifulsoup4-4.12.3-py3-none-any.whl.metadata (3.8 kB)
[2024-11-23T15:47:28.294+0000] {process_utils.py:194} INFO - Collecting soupsieve>1.2 (from beautifulsoup4>=4.12.3->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:28.302+0000] {process_utils.py:194} INFO -   Downloading soupsieve-2.6-py3-none-any.whl.metadata (4.6 kB)
[2024-11-23T15:47:28.416+0000] {process_utils.py:194} INFO - Collecting urllib3<3,>=1.26 (from urllib3[socks]<3,>=1.26->selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:28.425+0000] {process_utils.py:194} INFO -   Downloading urllib3-2.2.3-py3-none-any.whl.metadata (6.5 kB)
[2024-11-23T15:47:28.503+0000] {process_utils.py:194} INFO - Collecting trio~=0.17 (from selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:28.511+0000] {process_utils.py:194} INFO -   Downloading trio-0.27.0-py3-none-any.whl.metadata (8.6 kB)
[2024-11-23T15:47:28.582+0000] {process_utils.py:194} INFO - Collecting trio-websocket~=0.9 (from selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:28.590+0000] {process_utils.py:194} INFO -   Downloading trio_websocket-0.11.1-py3-none-any.whl.metadata (4.7 kB)
[2024-11-23T15:47:28.654+0000] {process_utils.py:194} INFO - Collecting certifi>=2021.10.8 (from selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:28.662+0000] {process_utils.py:194} INFO -   Downloading certifi-2024.8.30-py3-none-any.whl.metadata (2.2 kB)
[2024-11-23T15:47:28.732+0000] {process_utils.py:194} INFO - Collecting typing_extensions~=4.9 (from selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:28.740+0000] {process_utils.py:194} INFO -   Downloading typing_extensions-4.12.2-py3-none-any.whl.metadata (3.0 kB)
[2024-11-23T15:47:28.824+0000] {process_utils.py:194} INFO - Collecting websocket-client~=1.8 (from selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:28.836+0000] {process_utils.py:194} INFO -   Downloading websocket_client-1.8.0-py3-none-any.whl.metadata (8.0 kB)
[2024-11-23T15:47:28.923+0000] {process_utils.py:194} INFO - Collecting attrs>=23.2.0 (from trio~=0.17->selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:28.932+0000] {process_utils.py:194} INFO -   Downloading attrs-24.2.0-py3-none-any.whl.metadata (11 kB)
[2024-11-23T15:47:29.011+0000] {process_utils.py:194} INFO - Collecting sortedcontainers (from trio~=0.17->selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:29.020+0000] {process_utils.py:194} INFO -   Downloading sortedcontainers-2.4.0-py2.py3-none-any.whl.metadata (10 kB)
[2024-11-23T15:47:29.085+0000] {process_utils.py:194} INFO - Collecting idna (from trio~=0.17->selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:29.093+0000] {process_utils.py:194} INFO -   Downloading idna-3.10-py3-none-any.whl.metadata (10 kB)
[2024-11-23T15:47:29.155+0000] {process_utils.py:194} INFO - Collecting outcome (from trio~=0.17->selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:29.165+0000] {process_utils.py:194} INFO -   Downloading outcome-1.3.0.post0-py2.py3-none-any.whl.metadata (2.6 kB)
[2024-11-23T15:47:29.222+0000] {process_utils.py:194} INFO - Collecting sniffio>=1.3.0 (from trio~=0.17->selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:29.230+0000] {process_utils.py:194} INFO -   Downloading sniffio-1.3.1-py3-none-any.whl.metadata (3.9 kB)
[2024-11-23T15:47:29.301+0000] {process_utils.py:194} INFO - Collecting wsproto>=0.14 (from trio-websocket~=0.9->selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:29.311+0000] {process_utils.py:194} INFO -   Downloading wsproto-1.2.0-py3-none-any.whl.metadata (5.6 kB)
[2024-11-23T15:47:29.376+0000] {process_utils.py:194} INFO - Collecting pysocks!=1.5.7,<2.0,>=1.5.6 (from urllib3[socks]<3,>=1.26->selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:29.384+0000] {process_utils.py:194} INFO -   Downloading PySocks-1.7.1-py3-none-any.whl.metadata (13 kB)
[2024-11-23T15:47:29.447+0000] {process_utils.py:194} INFO - Collecting h11<1,>=0.9.0 (from wsproto>=0.14->trio-websocket~=0.9->selenium>=4.26.1->interpark==0.1.2->-r /tmp/venvnwnxf5ad/requirements.txt (line 1))
[2024-11-23T15:47:29.456+0000] {process_utils.py:194} INFO -   Downloading h11-0.14.0-py3-none-any.whl.metadata (8.2 kB)
[2024-11-23T15:47:29.501+0000] {process_utils.py:194} INFO - Downloading beautifulsoup4-4.12.3-py3-none-any.whl (147 kB)
[2024-11-23T15:47:29.547+0000] {process_utils.py:194} INFO - Downloading selenium-4.26.1-py3-none-any.whl (9.7 MB)
[2024-11-23T15:47:30.576+0000] {process_utils.py:194} INFO -    ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 9.7/9.7 MB 9.4 MB/s eta 0:00:00
[2024-11-23T15:47:30.584+0000] {process_utils.py:194} INFO - Downloading certifi-2024.8.30-py3-none-any.whl (167 kB)
[2024-11-23T15:47:30.623+0000] {process_utils.py:194} INFO - Downloading soupsieve-2.6-py3-none-any.whl (36 kB)
[2024-11-23T15:47:30.652+0000] {process_utils.py:194} INFO - Downloading trio-0.27.0-py3-none-any.whl (481 kB)
[2024-11-23T15:47:31.069+0000] {process_utils.py:194} INFO - Downloading trio_websocket-0.11.1-py3-none-any.whl (17 kB)
[2024-11-23T15:47:31.115+0000] {process_utils.py:194} INFO - Downloading typing_extensions-4.12.2-py3-none-any.whl (37 kB)
[2024-11-23T15:47:31.151+0000] {process_utils.py:194} INFO - Downloading urllib3-2.2.3-py3-none-any.whl (126 kB)
[2024-11-23T15:47:31.194+0000] {process_utils.py:194} INFO - Downloading websocket_client-1.8.0-py3-none-any.whl (58 kB)
[2024-11-23T15:47:31.233+0000] {process_utils.py:194} INFO - Downloading attrs-24.2.0-py3-none-any.whl (63 kB)
[2024-11-23T15:47:31.270+0000] {process_utils.py:194} INFO - Downloading PySocks-1.7.1-py3-none-any.whl (16 kB)
[2024-11-23T15:47:31.305+0000] {process_utils.py:194} INFO - Downloading sniffio-1.3.1-py3-none-any.whl (10 kB)
[2024-11-23T15:47:31.341+0000] {process_utils.py:194} INFO - Downloading wsproto-1.2.0-py3-none-any.whl (24 kB)
[2024-11-23T15:47:31.371+0000] {process_utils.py:194} INFO - Downloading idna-3.10-py3-none-any.whl (70 kB)
[2024-11-23T15:47:31.405+0000] {process_utils.py:194} INFO - Downloading outcome-1.3.0.post0-py2.py3-none-any.whl (10 kB)
[2024-11-23T15:47:31.433+0000] {process_utils.py:194} INFO - Downloading sortedcontainers-2.4.0-py2.py3-none-any.whl (29 kB)
[2024-11-23T15:47:31.466+0000] {process_utils.py:194} INFO - Downloading h11-0.14.0-py3-none-any.whl (58 kB)
[2024-11-23T15:47:31.533+0000] {process_utils.py:194} INFO - Building wheels for collected packages: interpark
[2024-11-23T15:47:31.538+0000] {process_utils.py:194} INFO -   Building wheel for interpark (pyproject.toml): started
[2024-11-23T15:47:32.490+0000] {process_utils.py:194} INFO -   Building wheel for interpark (pyproject.toml): finished with status 'done'
[2024-11-23T15:47:32.491+0000] {process_utils.py:194} INFO -   Created wheel for interpark: filename=interpark-0.1.2-py3-none-any.whl size=17701 sha256=f538a00f6e52ea075b4b73e859e5ac277ba0fad06adf3ccc1c3835ec704d44f3
[2024-11-23T15:47:32.492+0000] {process_utils.py:194} INFO -   Stored in directory: /tmp/pip-ephem-wheel-cache-7dewybr7/wheels/3b/e3/3f/2198c53e48ab1c477124eda1522aad636d2590cc0e6349feda
[2024-11-23T15:47:32.495+0000] {process_utils.py:194} INFO - Successfully built interpark
[2024-11-23T15:47:32.530+0000] {process_utils.py:194} INFO - Installing collected packages: sortedcontainers, websocket-client, urllib3, typing_extensions, soupsieve, sniffio, pysocks, idna, h11, certifi, attrs, wsproto, outcome, beautifulsoup4, trio, trio-websocket, selenium, interpark
[2024-11-23T15:47:36.159+0000] {process_utils.py:194} INFO - Successfully installed attrs-24.2.0 beautifulsoup4-4.12.3 certifi-2024.8.30 h11-0.14.0 idna-3.10 interpark-0.1.2 outcome-1.3.0.post0 pysocks-1.7.1 selenium-4.26.1 sniffio-1.3.1 sortedcontainers-2.4.0 soupsieve-2.6 trio-0.27.0 trio-websocket-0.11.1 typing_extensions-4.12.2 urllib3-2.2.3 websocket-client-1.8.0 wsproto-1.2.0
[2024-11-23T15:47:36.566+0000] {process_utils.py:186} INFO - Executing cmd: /tmp/venvnwnxf5ad/bin/python /tmp/venv-callitx9rckp/script.py /tmp/venv-callitx9rckp/script.in /tmp/venv-callitx9rckp/script.out /tmp/venv-callitx9rckp/string_args.txt /tmp/venv-callitx9rckp/termination.log
[2024-11-23T15:47:36.567+0000] {process_utils.py:190} INFO - Output:
[2024-11-23T15:47:37.920+0000] {process_utils.py:194} INFO - The chromedriver version (114.0.5735.90) detected in PATH at /usr/local/bin/chromedriver might not be compatible with the detected chrome version (131.0.6778.85); currently, chromedriver 131.0.6778.85 is recommended for chrome 131.*, so it is advised to delete the driver in PATH and retry
[2024-11-23T15:47:38.259+0000] {process_utils.py:194} INFO - Traceback (most recent call last):
[2024-11-23T15:47:38.260+0000] {process_utils.py:194} INFO -   File "/tmp/venv-callitx9rckp/script.py", line 48, in <module>
[2024-11-23T15:47:38.260+0000] {process_utils.py:194} INFO -     res = get_data(*arg_dict["args"], **arg_dict["kwargs"])
[2024-11-23T15:47:38.260+0000] {process_utils.py:194} INFO -           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-11-23T15:47:38.261+0000] {process_utils.py:194} INFO -   File "/tmp/venv-callitx9rckp/script.py", line 20, in get_data
[2024-11-23T15:47:38.261+0000] {process_utils.py:194} INFO -     get_data = extract_container_html()
[2024-11-23T15:47:38.262+0000] {process_utils.py:194} INFO -                ^^^^^^^^^^^^^^^^^^^^^^^^
[2024-11-23T15:47:38.262+0000] {process_utils.py:194} INFO -   File "/tmp/venvnwnxf5ad/lib/python3.12/site-packages/interpark/test.py", line 19, in extract_container_html
[2024-11-23T15:47:38.263+0000] {process_utils.py:194} INFO -     driver = webdriver.Chrome(options=chrome_options)
[2024-11-23T15:47:38.263+0000] {process_utils.py:194} INFO -              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-11-23T15:47:38.264+0000] {process_utils.py:194} INFO -   File "/tmp/venvnwnxf5ad/lib/python3.12/site-packages/selenium/webdriver/chrome/webdriver.py", line 45, in __init__
[2024-11-23T15:47:38.264+0000] {process_utils.py:194} INFO -     super().__init__(
[2024-11-23T15:47:38.264+0000] {process_utils.py:194} INFO -   File "/tmp/venvnwnxf5ad/lib/python3.12/site-packages/selenium/webdriver/chromium/webdriver.py", line 66, in __init__
[2024-11-23T15:47:38.265+0000] {process_utils.py:194} INFO -     super().__init__(command_executor=executor, options=options)
[2024-11-23T15:47:38.265+0000] {process_utils.py:194} INFO -   File "/tmp/venvnwnxf5ad/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py", line 238, in __init__
[2024-11-23T15:47:38.266+0000] {process_utils.py:194} INFO -     self.start_session(capabilities)
[2024-11-23T15:47:38.266+0000] {process_utils.py:194} INFO -   File "/tmp/venvnwnxf5ad/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py", line 325, in start_session
[2024-11-23T15:47:38.266+0000] {process_utils.py:194} INFO -     response = self.execute(Command.NEW_SESSION, caps)["value"]
[2024-11-23T15:47:38.267+0000] {process_utils.py:194} INFO -                ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
[2024-11-23T15:47:38.267+0000] {process_utils.py:194} INFO -   File "/tmp/venvnwnxf5ad/lib/python3.12/site-packages/selenium/webdriver/remote/webdriver.py", line 380, in execute
[2024-11-23T15:47:38.268+0000] {process_utils.py:194} INFO -     self.error_handler.check_response(response)
[2024-11-23T15:47:38.268+0000] {process_utils.py:194} INFO -   File "/tmp/venvnwnxf5ad/lib/python3.12/site-packages/selenium/webdriver/remote/errorhandler.py", line 229, in check_response
[2024-11-23T15:47:38.269+0000] {process_utils.py:194} INFO -     raise exception_class(message, screen, stacktrace)
[2024-11-23T15:47:38.269+0000] {process_utils.py:194} INFO - selenium.common.exceptions.WebDriverException: Message: unknown error: Chrome failed to start: exited abnormally.
[2024-11-23T15:47:38.269+0000] {process_utils.py:194} INFO -   (unknown error: DevToolsActivePort file doesn't exist)
[2024-11-23T15:47:38.270+0000] {process_utils.py:194} INFO -   (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
[2024-11-23T15:47:38.270+0000] {process_utils.py:194} INFO - Stacktrace:
[2024-11-23T15:47:38.270+0000] {process_utils.py:194} INFO - #0 0x5627a47ac4e3 <unknown>
[2024-11-23T15:47:38.271+0000] {process_utils.py:194} INFO - #1 0x5627a44dbc76 <unknown>
[2024-11-23T15:47:38.271+0000] {process_utils.py:194} INFO - #2 0x5627a4504d78 <unknown>
[2024-11-23T15:47:38.271+0000] {process_utils.py:194} INFO - #3 0x5627a4501029 <unknown>
[2024-11-23T15:47:38.272+0000] {process_utils.py:194} INFO - #4 0x5627a453fccc <unknown>
[2024-11-23T15:47:38.272+0000] {process_utils.py:194} INFO - #5 0x5627a453f47f <unknown>
[2024-11-23T15:47:38.272+0000] {process_utils.py:194} INFO - #6 0x5627a4536de3 <unknown>
[2024-11-23T15:47:38.272+0000] {process_utils.py:194} INFO - #7 0x5627a450c2dd <unknown>
[2024-11-23T15:47:38.273+0000] {process_utils.py:194} INFO - #8 0x5627a450d34e <unknown>
[2024-11-23T15:47:38.273+0000] {process_utils.py:194} INFO - #9 0x5627a476c3e4 <unknown>
[2024-11-23T15:47:38.273+0000] {process_utils.py:194} INFO - #10 0x5627a47703d7 <unknown>
[2024-11-23T15:47:38.274+0000] {process_utils.py:194} INFO - #11 0x5627a477ab20 <unknown>
[2024-11-23T15:47:38.274+0000] {process_utils.py:194} INFO - #12 0x5627a4771023 <unknown>
[2024-11-23T15:47:38.274+0000] {process_utils.py:194} INFO - #13 0x5627a473f1aa <unknown>
[2024-11-23T15:47:38.274+0000] {process_utils.py:194} INFO - #14 0x5627a47956b8 <unknown>
[2024-11-23T15:47:38.275+0000] {process_utils.py:194} INFO - #15 0x5627a4795847 <unknown>
[2024-11-23T15:47:38.275+0000] {process_utils.py:194} INFO - #16 0x5627a47a5243 <unknown>
[2024-11-23T15:47:38.275+0000] {process_utils.py:194} INFO - #17 0x7ff33d6e3144 <unknown>
[2024-11-23T15:47:38.276+0000] {process_utils.py:194} INFO - 
[2024-11-23T15:47:38.406+0000] {taskinstance.py:3311} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 505, in execute
    return super().execute(context=serializable_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 870, in execute_callable
    result = self._execute_python_callable_in_subprocess(python_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 588, in _execute_python_callable_in_subprocess
    raise AirflowException(error_msg) from None
airflow.exceptions.AirflowException: Process returned non-zero exit status 1.
Message: unknown error: Chrome failed to start: exited abnormally.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Stacktrace:
#0 0x5627a47ac4e3 <unknown>
#1 0x5627a44dbc76 <unknown>
#2 0x5627a4504d78 <unknown>
#3 0x5627a4501029 <unknown>
#4 0x5627a453fccc <unknown>
#5 0x5627a453f47f <unknown>
#6 0x5627a4536de3 <unknown>
#7 0x5627a450c2dd <unknown>
#8 0x5627a450d34e <unknown>
#9 0x5627a476c3e4 <unknown>
#10 0x5627a47703d7 <unknown>
#11 0x5627a477ab20 <unknown>
#12 0x5627a4771023 <unknown>
#13 0x5627a473f1aa <unknown>
#14 0x5627a47956b8 <unknown>
#15 0x5627a4795847 <unknown>
#16 0x5627a47a5243 <unknown>
#17 0x7ff33d6e3144 <unknown>

[2024-11-23T15:47:38.417+0000] {taskinstance.py:1225} INFO - Marking task as UP_FOR_RETRY. dag_id=interpark_to_s3, task_id=extract, run_id=scheduled__2024-11-21T00:00:00+00:00, execution_date=20241121T000000, start_date=20241123T154720, end_date=20241123T154738
[2024-11-23T15:47:38.445+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-23T15:47:38.446+0000] {standard_task_runner.py:124} ERROR - Failed to execute job 40 for task extract (Process returned non-zero exit status 1.
Message: unknown error: Chrome failed to start: exited abnormally.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Stacktrace:
#0 0x5627a47ac4e3 <unknown>
#1 0x5627a44dbc76 <unknown>
#2 0x5627a4504d78 <unknown>
#3 0x5627a4501029 <unknown>
#4 0x5627a453fccc <unknown>
#5 0x5627a453f47f <unknown>
#6 0x5627a4536de3 <unknown>
#7 0x5627a450c2dd <unknown>
#8 0x5627a450d34e <unknown>
#9 0x5627a476c3e4 <unknown>
#10 0x5627a47703d7 <unknown>
#11 0x5627a477ab20 <unknown>
#12 0x5627a4771023 <unknown>
#13 0x5627a473f1aa <unknown>
#14 0x5627a47956b8 <unknown>
#15 0x5627a4795847 <unknown>
#16 0x5627a47a5243 <unknown>
#17 0x7ff33d6e3144 <unknown>
; 77)
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py", line 117, in _start_by_fork
    ret = args.func(args, dag=self.dag)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/cli_config.py", line 49, in command
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/cli.py", line 116, in wrapper
    return f(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 483, in task_run
    task_return_code = _run_task_by_selected_method(args, _dag, ti)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 256, in _run_task_by_selected_method
    return _run_raw_task(args, ti)
           ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/cli/commands/task_command.py", line 341, in _run_raw_task
    return ti._run_raw_task(
           ^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/session.py", line 97, in wrapper
    return func(*args, session=session, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3005, in _run_raw_task
    return _run_raw_task(
           ^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 273, in _run_raw_task
    TaskInstance._execute_task_with_callbacks(
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3159, in _execute_task_with_callbacks
    result = self._execute_task(context, task_orig)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 3183, in _execute_task
    return _execute_task(self, context, task_orig)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 767, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 733, in _execute_callable
    return ExecutionCallableRunner(
           ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/utils/operator_helpers.py", line 252, in run
    return self.func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 505, in execute
    return super().execute(context=serializable_context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 417, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 238, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 870, in execute_callable
    result = self._execute_python_callable_in_subprocess(python_path)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 588, in _execute_python_callable_in_subprocess
    raise AirflowException(error_msg) from None
airflow.exceptions.AirflowException: Process returned non-zero exit status 1.
Message: unknown error: Chrome failed to start: exited abnormally.
  (unknown error: DevToolsActivePort file doesn't exist)
  (The process started from chrome location /usr/bin/google-chrome is no longer running, so ChromeDriver is assuming that Chrome has crashed.)
Stacktrace:
#0 0x5627a47ac4e3 <unknown>
#1 0x5627a44dbc76 <unknown>
#2 0x5627a4504d78 <unknown>
#3 0x5627a4501029 <unknown>
#4 0x5627a453fccc <unknown>
#5 0x5627a453f47f <unknown>
#6 0x5627a4536de3 <unknown>
#7 0x5627a450c2dd <unknown>
#8 0x5627a450d34e <unknown>
#9 0x5627a476c3e4 <unknown>
#10 0x5627a47703d7 <unknown>
#11 0x5627a477ab20 <unknown>
#12 0x5627a4771023 <unknown>
#13 0x5627a473f1aa <unknown>
#14 0x5627a47956b8 <unknown>
#15 0x5627a4795847 <unknown>
#16 0x5627a47a5243 <unknown>
#17 0x7ff33d6e3144 <unknown>

[2024-11-23T15:47:38.461+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 1
[2024-11-23T15:47:38.483+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-23T15:47:38.489+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
